benchmarks/source-agilitycms/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "agility-gatsby-starter",
+  "private": true,
+  "description": "A simplified bare-bones starter for Gatsby",
+  "version": "2.3.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby clean && gatsby build",
+    "develop": "gatsby develop",
+    "format": "prettier --write src/**/*.{js,jsx}",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\"",
+	"setup": "NODE_ENV=development node bin/setup.js",
+	"update": "NODE_ENV=development node bin/update.js"
```

Comment:
Please make this `data-update` to match the pattern used with other benchmarks, ex. https://github.com/gatsbyjs/gatsby/blob/947492971d33a508ace63f24c0be20870f3ae892/benchmarks/source-kontent/package.json#L13